app/helpers/turbo/frames_helper.rb

```diff
@@ -36,7 +36,7 @@ module Turbo::FramesHelper
   #   <%= turbo_frame_tag(Article.find(1), Comment.new) %>
   #   # => <turbo-frame id="article_1_new_comment"></turbo-frame>
   def turbo_frame_tag(*ids, src: nil, target: nil, **attributes, &block)
```

Comment:
Could you add an example to the documentation comment above like the one in https://github.com/hotwired/turbo-rails/pull/505/files#diff-7694cf5e712468388f0d32c6638e9d9508b612f0346ce4beb9093960a757fa59R26-R28?

Reply:
Yes, I've pulled your example in.